app/src/main/java/ch/epfl/sweng/eventmanager/ui/eventShowcase/fragments/schedule/ScheduleParentFragment.java

```diff
@@ -48,23 +56,88 @@ public void onCreate(Bundle savedInstanceState) {
         setRetainInstance(true);
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (scheduleViewModel == null) {
+            scheduleViewModel = ViewModelProviders.of(requireActivity()).get(ScheduleViewModel.class);
+        }
+
+        Log.i(TAG, "Resumed model from memory : " + scheduleViewModel);
```

Comment:
Sure you want to keep that log ?

Reply:
I've put this line to be coherent with the other classes of the project ! If I remove it i think we should also remove it in every other classes ...

Reply:
Ok let it be then